superdesk/io/feeding_services/ftp.py

```diff
@@ -79,8 +97,14 @@ def _update(self, provider, update):
             except KeyError:
                 logger.warning('missing move_path')
                 move_dest_path = None
+            try:
+                move_dest_path_error = os.path.join(config.get('path', ''), config['move_path_error'])
```

Comment:
I've pinged @karelpetrak about this, he rather like the default directory to be consistent with files, so same opinion as @petrjasek . So I'll go this way and change the P.R. later today.